packages/cli/src/databases/sqlite/migrations/1636626154932-CreateUserManagement.ts

```diff
@@ -1,78 +1,96 @@
+		const instanceOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
 
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("member", "global");
 		`);
 
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("owner", "workflow");
 		`);
 
+		const workflowOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
+
 		await queryRunner.query(`
-			INSERT INTO role (name, scope)
+			INSERT INTO "${tablePrefix}role" (name, scope)
 			VALUES ("owner", "credential");
 		`);
 
+		const credentialOwnerRole = await queryRunner.query('SELECT last_insert_rowid() as insertId');
+
+		const survey = loadSurveyFromDisk();
+
+		const ownerUserId = uuid();
+		await queryRunner.query(`
+			INSERT INTO "${tablePrefix}user" (id, globalRoleId, personalizationAnswers) values
+			(?, ?, ?)
+		`, [ownerUserId, instanceOwnerRole[0].insertId, survey ?? null]);
+
+		await queryRunner.query(`
+			INSERT INTO "${tablePrefix}shared_workflow" (createdAt, updatedAt, roleId, userId, workflowId)
+			select DATETIME('now'), DATETIME('now'), '${workflowOwnerRole[0].insertId}', '${ownerUserId}', id from "${tablePrefix}workflow_entity"
+		`);
+
 		await queryRunner.query(`
-			INSERT INTO user(id, globalRoleId) values
-			('${uuid()}', ${insertedOwnerRole[0].insertId})
+			INSERT INTO "${tablePrefix}shared_credentials" (createdAt, updatedAt, roleId, userId, credentialsId)
+			select DATETIME('now'), DATETIME('now'), '${credentialOwnerRole[0].insertId}', '${ownerUserId}', id from "${tablePrefix}credentials_entity"
 		`);
 
 		await queryRunner.query(`
-			INSERT INTO settings(key, value, loadOnStartup) values
+			INSERT INTO "${tablePrefix}settings" (key, value, loadOnStartup) values
 			('userManagement.hasOwner', 'false', true)
 		`);
 	}
 
 	public async down(queryRunner: QueryRunner): Promise<void> {
-		await queryRunner.query(`DROP TABLE "shared_credentials"`);
-		await queryRunner.query(`DROP TABLE "shared_workflow"`);
-		await queryRunner.query(`DROP INDEX "IDX_e12875dfb3b1d92d7d7c5377e2"`);
-		await queryRunner.query(`DROP TABLE "user"`);
-		await queryRunner.query(`DROP TABLE "role"`);
-		await queryRunner.query(`DROP TABLE "settings"`);
-		await queryRunner.query(`DROP INDEX "IDX_xeendlvptc5jy4hbol17b5xery"`);
+		const tablePrefix = config.get('database.tablePrefix');
+		await queryRunner.query(`DROP TABLE "${tablePrefix}shared_credentials"`);
+		await queryRunner.query(`DROP TABLE "${tablePrefix}shared_workflow"`);
+		await queryRunner.query(`DROP INDEX "IDX_${tablePrefix}e12875dfb3b1d92d7d7c5377e2"`);
+		await queryRunner.query(`DROP TABLE "${tablePrefix}user"`);
+		await queryRunner.query(`DROP TABLE "${tablePrefix}role"`);
+		await queryRunner.query(`DROP TABLE "${tablePrefix}settings"`);
```

Comment:
we need to add to drop the index on `execution_entity`

Reply:
This was an error in the versioning; we did not change existing tables.

Tested and dropping the tables also removes their indexes so for the sake of simplicity, only tables are being dropped.